src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="/images/imac.jpeg"
+        alt="APPLE iMac 27"
+       >
+
+      <span class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
```

Comment:
Missed space between sibling elements
```suggestion
      <span class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
      
      <span class="card__code" >Product code: 195434</span>
```